### PR TITLE
hotfix/SC-5948 - Move FEATURE_MULTI_LOGIN_INSTANCES env to schema file

### PR DESCRIPTION
**Pull request by mwolowyk-cap**
# Description

Hotfix for https://ticketsystem.schul-cloud.org/browse/SC-5948

## Links to Tickets or other pull requests

https://ticketsystem.schul-cloud.org/browse/SC-5948

## Approval for review
- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definition of Done
More and detailed information on the *definition of done* can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

File: controllers/login.js [hunk 1]
```diff
@@ -122,7 +121,7 @@ router.get('/loginRedirect', (req, res, next) => {
 		if (isAuthenticated) {
 			return redirectAuthenticated(req, res);
 		}
-		if (FEATURE_MULTI_LOGIN_INSTANCES) {
+		if (Configuration.get('FEATURE_MULTI_LOGIN_INSTANCES')) {
```
Comment (tofixx) [inline controllers/login.js:124]:
```suggestion
		if (Configuration.get('FEATURE_MULTI_LOGIN_INSTANCES') === true) {
```
i would explictly use the existing type here

File: config/default.schema.json [hunk 2]
```diff
@@ -313,6 +313,11 @@
 			"type": "string",
 			"default": "",
 			"description": "The URL to Confluence. Without a link, there are subpages in the side menu and on the /help page. With the link the option from the Hilfebereich side menu leads to confluence."
+		},
+		"FEATURE_MULTI_LOGIN_INSTANCES": {
```
Comment (tofixx) [inline config/default.schema.json:317]: please, name feature flags `FEATURE...ENABLED`

File: controllers/login.js (hunk 1 above)
Comment (janrenz) [reply controllers/login.js:124]: @tofixx This could be moved to a helper in the Configuration Class

**Comment by BlackGhostHD** [comment]
@mwolowyk-cap Why was this merged into 24.0?
@CordlessWool I thought the release was already completed?

**Comment by BlackGhostHD** [comment]
These changes are reflected in https://github.com/hpi-schul-cloud/schulcloud-client/pull/2068 and published as hotfix 24.0.3.

**Pull request merged**